src/Waives.Reactive/WaivesApi.cs

```diff
@@ -125,12 +125,16 @@ public static async Task<WaivesClient> Login(string clientId, string clientSecre
         /// }
         /// ]]>
         /// </example>
+        /// <param name="clearOrphans">If set to <c>true</c>, it will (immediately) delete all documents
+        /// in existence in the Waives account; if set to <c>false</c>, no such clean up will be completed.
+        /// Defaults to <c>true</c>.</param>
         /// <returns>A new <see cref="Pipeline"/> instance with which you can
         /// configure your document processing pipeline.</returns>
-        public static Pipeline CreatePipeline()
+        public static Pipeline CreatePipeline(bool clearOrphans = true)
```

Comment:
✋ I suggest we call this option `deleteExistingDocuments` rather than `clearOrphans`. Otherwise we're introducing new terminology that we don't use anywhere else.